app/modals/stacking/stacking-modal.tsx

```diff
@@ -93,6 +93,11 @@ export const StackingModal: FC<StackingModalProps> = props => {
     balance: selectAddressBalance(state),
   }));
 
+  const accountBalance = new BigNumber(balance?.balance ?? 0);
+
+  const shouldModifyTxFee =
```

Comment:
I think I missed some context here, why 
> or within 1000 usTX of the address' balance. 
?

Reply:
You can't stack 100% of your balance as you need to pay the fee.

Reply:
The fee rate in stacks-node is hard-coded per byte at the moment. So, until that changes, the fee will always be 260 ustx. Maybe we should just stick to that. If the fee rates change in stacks-node, then this adjustment might not even be enough, and we'll want to revisit this regardless of if 1000 is enough.

Reply:
Done